src/Containers/PaletteForm/PaletteForm.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { addPalette } from '../../actions/actions';
+import './PaletteForm.scss';
+
+const PaletteForm = (props) => {
+
+  const [ project, selectProject ] = useState('');
```

Comment:
Nice work using hooks to make the control form - that does look a lot cleaner then a stateful component!